main.py

```diff
@@ -303,6 +332,9 @@ def run_selected_trip():
 
         last_color = current_color
 
-vrooom()
```

Comment:
@FortressBen @EricGamer1127 the code you push to master should always call vroom, not the test trip